dist/ami/scylla.json

```diff
@@ -69,13 +75,13 @@
   ],
   "provisioners": [
     {
-      "destination": "/home/{{user `ssh_username`}}/",
+      "destination": "/home/ec2-user/",
       "source": "files/",
       "type": "file"
     },
     {
       "inline": [
-        "sudo /home/{{user `ssh_username`}}/scylla_install_ami {{ user `install_args` }}"
+        "sudo /home/ec2-user/scylla_install_ami {{ user `install_args` }}"
```

Comment:
better use a parameter so that it can be controlled in a single place

Reply:
reverted the change to keep username variable in a single place.